sql/parser/drop.go

```diff
@@ -53,11 +53,6 @@ func (node *DropTable) String() string {
 	if node.IfExists {
 		_, _ = buf.WriteString("IF EXISTS ")
 	}
-	for i, n := range node.Names {
-		if i > 0 {
-			_, _ = buf.WriteString(", ")
-		}
-		_, _ = buf.WriteString(n.String())
-	}
+	_, _ = buf.WriteString(node.Names.String())
```

Comment:
Not sure it is worth it. I'm ignoring the error because one is never returned for `bytes.Buffer.WriteString`: https://github.com/golang/go/blob/master/src/bytes/buffer.go#L141